neuralmagic/benchmarks/scripts/benchmark_result.py

```diff
@@ -0,0 +1,147 @@
+"""
+Defines a BenchmarkResult class that all the benchmarks use store the benchmark results.
+"""
+
+import json
+
+from vllm import __version__ as __vllm_version__
+from typing import Optional
+from types import SimpleNamespace
+from pathlib import Path
+from .common import get_benchmarking_context
+from datetime import datetime
+from dataclasses import dataclass, field
+from enum import Enum
+
+# NOTE - PLEASE READ:
```

Comment:
will this error or act weird when there is a mismatch?

Reply:
No. I added the JSON version for the Elastic Search usecases - So we know what "keys" are guaranteed to exist under some version.